index.js

```diff
@@ -0,0 +1,163 @@
+    
+    playerObj.goals.forEach(function(goal) {
+        const li = document.createElement('li')
+        const iframe = document.createElement('iframe')
+
+        iframe.src = goal.link
+        iframe.width = "420" 
+        iframe.height = "315"
+        li.textContent = goal.description
+        
+        playerGoals.append(iframe)
+        playerGoals.append(li)
+        
+    })
+
+
+
+}
+
+/******* Deliverable 2 *******/
+// When a user clicks the like button for a player, that player's likes should increase by 1. The player's updated likes should persist on the server, and also be displayed on the page.
+
+/*********** DOM Elements *********/
+
+/*********** Event Listener *********/
+likesButton.addEventListener("click", updateLikes)
+/*********** FETCH *********/
+function updateLikes(){
+
+fetch("http://localhost:3000/players/1", {
+method: 'PATCH',
+headers: {
+    'Content-Type': 'application/json',
+},
+body: JSON.stringify({
+    likes: parseInt(playerLikes.textContent) + 1
+    
+}),
+})
+.then(response => response.json())
+.then(data => {
+    console.log(data)
+    playerLikes.textContent = data.likes  
+      
+})
+}
+
+/********** Event Handler ********/
+// function incrementLike(data){
+//     const likes = parseInt(data) + 1
+//     data  = likes
+// }
+
+
+/********** Initial Render ********/
+document.addEventListener("DOMContentLoaded",function(){
+    fetch("http://localhost:3000/players/1")
+.then(response => response.json())
+.then(data => renderPlayer(data))
+})
+
+/********** Deliverable 3 ********/
+/*When a user submits the form**, a new goal video should be displayed for the player. The new goal video should also persist in the backend, so when you refresh the page, you can still see the new goal video.*/
+//goalForm
+//goalList
+// function addGoals (event) {
+// event.preventDefault()
+
+// const goalObj = {
+//    playerId: 1,
+//    link: event.target.value
+// }
+
+
+// const postForm = function(event) {
+//     fetch("http://localhost:3000/goals", {
+// method: 'POST',
+// headers: {
+// 'Content-Type': "application/json",
+// },
+// body: JSON.stringify({
+//         playerId: 1,
+//         link: event.target.link.value,
+//         description: event.target.description.value
+
+// }),
+// })
+
+
+//     .then(response => response.json())
+//     .then(data => {console.log("Success", data)})
+// }
+
+/********** Event Listener ********/
+goalForm.addEventListener("submit", handleGoalFormSubmit)
+
+/********** Render Form ********/
+
+function renderGoals (goalObj) {
+    const iframe = document.createElement('iframe')
+    const li = document.createElement('li')
+
+    iframe.src = goalObj.link
+    li.textContent = goalObj.description
+
+    goalList.append(iframe)
+    goalList.append(li)
+}
+
+function handleGoalFormSubmit (event) {
+    event.preventDefault()
+
+    const goalObj = {
+    link: event.target.link.value,
+    description: event.target.description.value
+}
```

Comment:
This won't quite work, when you refresh the page the goals won't show up for Mo. **Make sure to read the API documentation to see what you need to send in the body of the request**. In this case, we'd also need the `playerId` in order to associate the goal with the player.